kafka/modules/kafka/2.1.1/install.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
Yes, this file is duplicated, the module doesn't let us reference scripts outside the module so we need the same install script for each kafka module

Reply:
Because it's a shell script, did you already try to do something like `. /common-script.sh` at the top and see if it works? Having two different scripts but using the same `common-script-sh` file.

Reply:
Argh! I didn't think of that, but that should work, let me try

Reply:
After trying this now, I remember trying this before, unfortunately we can't do this with these particular scripts since they are used at build time instead of runtime. The modules are run one at a time and isolated from one another so I can't reference build scripts outside of the modules itself